build/bakefiles/files.bkl

```diff
@@ -2285,8 +2285,8 @@ IMPORTANT: please read docs/tech/tn0016.txt before modifying this file!
 <set var="MSW_RSC" hints="files">
     <!-- Resources must be installed together with headers: -->
     wx/msw/wx.manifest
-    wx/msw/amd64.manifest
-    wx/msw/ia64.manifest
+    wx/msw/wx_dpi_aware.manifest
```

Comment:
The two DPI-aware manifests were not included here and in `build/cmake/files.cmake`. I asssumed it was not on purpose but by mistake, so I added them here and there.

Reply:
Yes, it must have been by mistake, thanks for fixing this. And `build/files` is the primary source, run `./build/upmake` to update the other ones from it.